### PR TITLE
Adding tobiko image

### PR DESCRIPTION
**Pull request by eduolivares**
This PR adds the files needed to create a tobiko container image.
Co-Authored-By: Omer Schwartz

### CODE REVIEW

**Comment by rabi** [comment]
Looks good to me, can you remove the merge commit? https://github.com/openstack-k8s-operators/tcib/pull/88/commits/09091dbb266ec0f8b5afa68434a68f376c5b4f10

File: container-images/tcib/base/os/tobiko/run_tobiko.sh [hunk 1]
```diff
@@ -2,12 +2,34 @@
...
+    cp $TOBIKO_DIR/external_files/clouds.yaml $HOME/.config/openstack/
+fi
+
 pushd ${TOBIKO_DIR}
 
 export OS_CLOUD=default
 echo "export OS_CLOUD=default" > .bashrc
 
+mkdir -p .downloaded-images
+curl https://cloud-images.ubuntu.com/minimal/releases/jammy/release/ubuntu-22.04-minimal-cloudimg-amd64.img -o .downloaded-images/ubuntu-minimal
```
Comment (rabi) [inline container-images/tcib/base/os/tobiko/run_tobiko.sh:16]: Should not ubuntu version be configurable?

Comment (rabi) [reply container-images/tcib/base/os/tobiko/run_tobiko.sh:16]: Also can't be this be fedora or something?

File: container-images/tcib/base/os/tobiko/tobiko.yaml [hunk 2]
```diff
@@ -7,10 +7,8 @@ tcib_actions:
 - run: chmod 440 /etc/sudoers.d/tobiko_sudoers
 - run: mkdir -p /var/lib/tempest/external_files
 - run: chown -R tobiko.tobiko /var/lib/tobiko
-- run: git clone https://opendev.org/x/tobiko /var/lib/tobiko
-  # The following line doesn't work
-  #- run: cat {{ tobiko_config }} >> /var/lib/tobiko/tobiko.conf
-- run: python3 -m pip install --upgrade pip tox
+- run: python3 -m pip install --upgrade pip
+- run: python3 -m pip install 'tox>=3.8,<4.0'
```
Comment (rabi) [inline container-images/tcib/base/os/tobiko/tobiko.yaml:11]: Was about to question this but noticed https://opendev.org/x/tobiko/src/branch/master/roles/tobiko-ensure-tox/defaults/main.yaml

File: container-images/tcib/base/os/tobiko/run_tobiko.sh [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+[ubuntu]
+interface_name = enp3s0
+image_file = /var/lib/tobiko/.downloaded-images/ubuntu-minimal
+
+[keystone]
+interface = public' > tobiko.conf
+
+python3 -m tox -e py3 --notest
+sleep 9999999
```
Comment (rabi) [inline container-images/tcib/base/os/tobiko/run_tobiko.sh:34]: Why this sleep here? You can also run tox -e py3,scenario ?

Comment (eduolivares) [reply container-images/tcib/base/os/tobiko/run_tobiko.sh]: Done.
I did something wrong when I removed the merge commit:
https://github.com/openstack-k8s-operators/tcib/compare/09091dbb266ec0f8b5afa68434a68f376c5b4f10..43e43d93c18255163da55fadb26bba6f31664881

File: container-images/tcib/base/os/tobiko/run_tobiko.sh (hunk 1 above)
Comment (eduolivares) [reply container-images/tcib/base/os/tobiko/run_tobiko.sh]: Now it is configurable.
Tobiko uses Ubuntu images as advanced images at this moment:
https://github.com/redhat-openstack/tobiko/blob/master/tobiko/openstack/stacks/_ubuntu.py

**Review by rabi — APPROVED** [review]
/lgtm

**Pull request merged**